ext/ddtrace_profiling_native_extension/libdatadog_helpers.h

```diff
@@ -34,3 +34,7 @@ size_t read_ddogerr_string_and_drop(ddog_Error *error, char *string, size_t capa
 // ruby_value_type that Ruby uses so that we can also use this for debugging.
 const char *ruby_value_type_to_string(enum ruby_value_type type);
 ddog_CharSlice ruby_value_type_to_char_slice(enum ruby_value_type type);
+
+// Returns a dynamically allocated string from the provided char slice.
+// WARN: The returned string must be explicitly freed with ruby_xfree.
+char* string_from_char_slice(ddog_CharSlice slice);
```

Comment:
Minor: Since the implementation is so trivial, should we just move it here and make it `inline static`?